src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DraftReferralService.kt

```diff
@@ -439,6 +443,7 @@ class DraftReferralService(
 
     val sentReferral = referralRepository.save(referral)
     createReferralLocation(draftReferral, sentReferral)
+    eventPublisher.custodyLocationLookupEvent(sentReferral)
     eventPublisher.referralSentEvent(sentReferral)
```

Comment:
As far as I know, the generic advice around events are:

- keep the number of events small as possible
- model events as "something important happened" instead of "do this" (makes reasoning about behaviour simple)

Could we do the lookup as a consequence of the "referral sent" event?

Reply:
Good point, that makes a lot more sense :) I've removed the CustodyLocationLookupEvent and updated the listener to listen to "referral sent" events instead